### PR TITLE
Update bootstrap italia to version 2.8.7

### PR DESCRIPTION
**Pull request by r1gm**

### CODE REVIEW

File: bootstrap-themes/pom.xml [hunk 1]
```diff
@@ -40,16 +39,21 @@
...
+            <exclusions>
+                <exclusion>  <!-- exclude dependency, because causing dependency conflict -->
+                    <groupId>org.webjars.npm</groupId>
+                    <artifactId>video.js</artifactId>
+                </exclusion>                
+                <exclusion>  <!-- exclude dependency, no versions availablet -->
+                    <groupId>org.webjars.npm</groupId>
+                    <artifactId>design-tokens-italia</artifactId>
+                </exclusion>    
+            </exclusions>
```
Comment (r1gm) [inline bootstrap-themes/pom.xml:58]: Should we exclude bootstrap also ?, forcing bootstrap version using current wicket-bootstrap webjar version.

Comment (martin-g) [reply bootstrap-themes/pom.xml:58]: Yes, I think this is a good idea!

File: bootstrap-themes/pom.xml [hunk 2]
```diff
@@ -40,16 +39,21 @@
         </dependency>
 
         <dependency>
-            <groupId>org.webjars</groupId>
+            <groupId>org.webjars.npm</groupId>
             <artifactId>bootstrap-italia</artifactId>
             <version>${bootstrap-italia.version}</version>
+            <exclusions>
+                <exclusion>  <!-- exclude dependency, because causing dependency conflict -->
```
Comment (martin-g) [inline bootstrap-themes/pom.xml:50]: What kind of conflict does it cause ?

File: bootstrap-themes/pom.xml [hunk 3]
```diff
@@ -40,16 +39,21 @@
...
+                    <groupId>org.webjars.npm</groupId>
+                    <artifactId>video.js</artifactId>
+                </exclusion>                
+                <exclusion>  <!-- exclude dependency, no versions available -->
+                    <groupId>org.webjars.npm</groupId>
+                    <artifactId>design-tokens-italia</artifactId>
```
Comment (martin-g) [inline bootstrap-themes/pom.xml:56]: How `<artifactId>bootstrap-italia</artifactId>` got promoted to Maven Central if its dependencies are not available ?!

File: bootstrap-themes/pom.xml (hunk 2 above)
Comment (r1gm) [reply bootstrap-themes/pom.xml:50]: Output : 

Failed to execute goal on project wicket-bootstrap-themes: Could not resolve dependencies for project de.agilecoders.wicket:wicket-bootstrap-themes:bundle:7.0.7-SNAPSHOT: Failed to collect dependencies for de.agilecoders.wicket:wicket-bootstrap-themes:jar:7.0.7-SNAPSHOT: Could not resolve version conflict among [org.webjars.npm:bootstrap-italia:jar:2.8.7 -> org.webjars.npm:video.js:jar:[8.0.4,9.0.0-0) -> org.webjars.npm:m3u8-parser:jar:[7.1.0,8.0.0-0) -> org.webjars.npm:videojs__vhs-utils:jar:[3.0.5,4), org.webjars.npm:bootstrap-italia:jar:2.8.7 -> org.webjars.npm:video.js:jar:[8.0.4,9.0.0-0) -> org.webjars.npm:mpd-parser:jar:[1.2.2,2.0.0-0) -> org.webjars.npm:videojs__vhs-utils:jar:[4.0.0,5.0.0-0), org.webjars.npm:bootstrap-italia:jar:2.8.7 -> org.webjars.npm:video.js:jar:[8.0.4,9.0.0-0) -> org.webjars.npm:videojs__vhs-utils:jar:[4.0.0,5.0.0-0), org.webjars.npm:bootstrap-italia:jar:2.8.7 -> org.webjars.npm:video.js:jar:[8.0.4,9.0.0-0) -> org.webjars.npm:videojs__http-streaming:jar:3.13.1 -> org.webjars.npm:videojs__vhs-utils:jar:4.0.0, org.webjars.npm:bootstrap-italia:jar:2.8.7 -> org.webjars.npm:video.js:jar:[8.0.4,9.0.0-0) -> org.webjars.npm:aes-decrypter:jar:[4.0.1,5.0.0-0) -> org.webjars.npm:videojs__vhs-utils:jar:[3.0.5,4)] -> [Help 1]

File: bootstrap-themes/pom.xml (hunk 3 above)
Comment (r1gm) [reply bootstrap-themes/pom.xml:56]: Magic ? 

Output :
```
Failed to execute goal on project wicket-bootstrap-themes: Could not resolve dependencies for project
 de.agilecoders.wicket:wicket-bootstrap-themes:bundle:7.0.7-SNAPSHOT: Failed to collect dependencies at 
org.webjars.npm:bootstrap-italia:jar:2.8.7 -> org.webjars.npm:design-tokens-italia:jar:[1.1.1,2.0.0-0): No versions 
available for org.webjars.npm:design-tokens-italia:jar:[1.1.1,2.0.0-0) within specified range -> [Help 1]
```

Try to add new webjar from https://www.webjars.org/ but failed

Deploy Log

```
Starting Deploy
The metadata was missing a required field: /repository/url
```

issue report created
https://github.com/italia/design-tokens-italia/issues/5

File: bootstrap-themes/pom.xml (hunk 1 above)
Comment (r1gm) [reply bootstrap-themes/pom.xml:58]: done

**Comment by martin-g** [comment]
Thank you, @r1gm !

**Pull request merged**